### PR TITLE
Update OpenZWave versions

### PR DESCRIPTION
**Pull request by cgarwood**
Updates versions of OpenZWave, qt-openzwave, and ozw-admin

OpenZWave changes: https://github.com/OpenZWave/open-zwave/compare/a35732f107e3620578c1cb9f027e0a71b5410697...0dba812c2568a59e1cbe9f06c84b5f1adbda3ccb

qt-openzwave changes: https://github.com/OpenZWave/qt-openzwave/compare/337e48807568e4e7f90d3b8441cc126c9e057cde...3ad9138f40a856366a7d85fb110b1ed91e1534a0

ozw-admin changes: https://github.com/OpenZWave/ozw-admin/compare/278427be5d52a04aaab8e9d767f7e9a210bc1b58...da04ebfbadc57484c487ebc846431eb283e83176

### CODE REVIEW

File: zwave/Dockerfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ RUN \
         pwgen \
         qt5-qtbase \
         qt5-qtremoteobjects \
+        qt5-qtserialport \
```
Comment (MartinHjelmare) [inline zwave/Dockerfile:23]: What does this change mean?

Comment (cgarwood) [reply zwave/Dockerfile:23]: It's a dependency that was added to qt-openzwave, required for it to compile.

**Comment by frenck** [comment]
@cgarwood Can we bump the add-on to Alpine 3.12, while you are at it?

**Comment by cgarwood** [comment]
@frenck is that just changing the `build_from` images from arch-base:3.11 to arch-base:3.12?

**Comment by pvizeli** [comment]
yes and update the version of QT libs from 5.12 to 5.14

**Comment by cgarwood** [comment]
Updated the build.json and kicked off a build on my home system to test it out.

**Comment by cgarwood** [comment]
Built OK on my system and did a test run for a few hours.

Also pulled in a new commit of OpenZWave. Updated OpenZWave changelog: https://github.com/OpenZWave/open-zwave/compare/a35732f107e3620578c1cb9f027e0a71b5410697...5d3978d5cac8d1bf556bd20c56ec2309b7cd7091

**Comment by frenck** [comment]
Nice job there @cgarwood 👍

**Pull request merged**